iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -667,6 +667,21 @@ - (void)updateBookmark:(MWMMarkID)bookmarkId
     bookmark->SetCustomName(title.UTF8String);
 }
 
+- (void)updateBookmark:(MWMMarkID)bookmarkId
+              setColor:(MWMBookmarkColor)color {
+  auto editSession = self.bm.GetEditSession();
+
+  auto bookmark = editSession.GetBookmarkForEdit(bookmarkId);
+  if (!bookmark)
```

Comment:
Let's crash here.

Reply:
Will it work in release? Is there a better way to crash and show some error to a user?

```

Assertions are disabled if the preprocessor macro NS_BLOCK_ASSERTIONS is defined or the ENABLE_NS_ASSERTIONS Xcode build setting is disabled.
```

Reply:
Should we show this internal error from the core to the user? And what message should we use?
There are no exception at all in the BookmarksManager.

Reply:
Ok, I see that C++ core never checks for nullptr after getting bookmarks using the same method. So let's replace if here and in other similar places with
NSAssert(bookmark, @"Invalid bookmark id: @%llu", bookmarkId);

Reply:
done!